beefy-gadget/src/lib.rs

```diff
@@ -165,7 +169,7 @@ struct BeefyWorker<Block: BlockT, Id, Signature, FinalityNotifications> {
 	local_id: Id,
 	key_store: SyncCryptoStorePtr,
 	min_interval: u32,
-	rounds: Rounds<Block::Hash, NumberFor<Block>, Id, Signature>,
+	rounds: Rounds<H256, NumberFor<Block>, Id, Signature>,
```

Comment:
Could we do `type MmrRootHash = H256;`? It took me a while to figure out what that `H256` here is.